source/jormungandr/jormungandr/interfaces/v1/serializer/places.py

```diff
@@ -204,42 +148,33 @@ def get_poi(self, obj):
 class AddressSerializer(serpy.DictSerializer):
     id = CoordId()
     coord = CoordField()
-    house_number = serpy.MethodField()
-    label = PropertyPathField(attr='properties.geocoding.label')
-    name = PropertyPathField(attr='properties.geocoding.name')
+    house_number = IntNestedPropertyField(attr='properties.geocoding.housenumber')
```

Comment:
hum I think for this one you need the Method find because we strip the `bis`, `ter`, ... no ?